docs/articles/documentation/examples/README.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: docs
+title: Examples
+permalink: /documentation/examples/
+---
+# Examples
+
+This section lists examples from the [testcafe-examples](https://github.com/DevExpress/testcafe-examples) repository. All examples are ready to run.
+
+## Locating Elements
+
+Example | Description
+------- | ------------
+[Find Element By Trimmed Text](https://github.com/DevExpress/testcafe-examples/tree/master/examples/find-element-by-trimmed-text) | Find a header element with extra space characters by trimmed text content.
+[Use XPath Selectors](https://github.com/DevExpress/testcafe-examples/tree/master/examples/use-xpath-selectors) | Find an input with an XPath selector.
```

Comment:
Can you clarify what you mean by "an input with an XPath selector"? Does this feature search for XPath expressions within markup? If not, please clarify.

Reply:
It means the example uses an XPath selector to find an input (I just didn't want to repeat the first column).

Reply:
```suggestion
[Use XPath Selectors](https://github.com/DevExpress/testcafe-examples/tree/master/examples/use-xpath-selectors) | Find an input with an XPath selector.
```

Reply:
Thank you. Okay, your version is fine then.